diff --git a/index.d.ts b/index.d.ts
index 346d0b2d72420a5eb16b56f0cc79476eee48b6f2..ad88746619ca616a4abc5f55bfd84f356ed1955a 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -19,7 +19,7 @@
 import FormData = require('form-data');
 import { RequestOptions } from "http";
 import { URLSearchParams, URL } from "url";
-import { AbortSignal } from "./externals";
+// import { AbortSignal } from "./externals";
 
 export class Request extends Body {
     constructor(input: RequestInfo, init?: RequestInit);
